src/net/rpc.rs

```diff
@@ -4137,6 +4349,48 @@ mod test {
                         .to_account_principal(),
                     None,
                     false,
+                    true,
+                )
+            },
+            |ref http_request, ref http_response, ref mut peer_client, ref mut peer_server| {
+                let req_md = http_request.metadata().clone();
+                match http_response {
+                    HttpResponseType::GetAccount(response_md, data) => {
+                        assert_eq!(data.nonce, 4);
+                        let balance = u128::from_str_radix(&data.balance[2..], 16).unwrap();
+                        assert_eq!(balance, 999999877);
+                        true
+                    }
+                    _ => {
+                        error!("Invalid response; {:?}", &http_response);
+                        false
+                    }
+                }
+            },
+        );
+    }
+
+    #[test]
+    #[ignore]
+    fn test_rpc_get_account_use_latest_tip_no_microblocks() {
+        test_rpc(
+            "test_rpc_get_account",
+            40110,
+            40111,
+            50110,
+            50111,
+            false,
+            |ref mut peer_client,
+             ref mut convo_client,
+             ref mut peer_server,
+             ref mut convo_server| {
+                convo_client.new_getaccount(
+                    StacksAddress::from_string("ST2DS4MSWSGJ3W9FBC6BVT0Y92S345HY8N3T6AV7R")
+                        .unwrap()
+                        .to_account_principal(),
+                    None,
+                    false,
+                    true,
```

Comment:
Can we have a few comments about why we expect the answers in the tests to be the way they are? And why there are not expected to be what they are not?

Reply:
Added comment.

Reply:
this is a very helpful comment!, but can you please do the same for the other tests you are adding?

Reply:
nice comments, thank you!